test/reporting/test_ssh_bruter_autoreporter_integration.py

```diff
@@ -0,0 +1,68 @@
+import tempfile
+from pathlib import Path
+from test.base import ArtemisModuleTestCase
+
+from jinja2 import BaseLoader, Environment, StrictUndefined
+from karton.core import Task
+
+from artemis.binds import Service, TaskType
+from artemis.modules.ssh_bruter import SSHBruter
+from artemis.reporting.base.language import Language
+from artemis.reporting.base.reporters import reports_from_task_result
+from artemis.reporting.base.templating import build_message_template
+from artemis.reporting.export.translations import install_translations
+
+environment = Environment(
```

Comment:
IMO if we have multiple reporting test, we may factor the common code into something called BaseReportingTest, so that lines 15-22 don't duplicate with bruter